packaging/obs/grafana-ha-cluster-dashboards/grafana-ha-cluster-dashboards.spec

```diff
@@ -0,0 +1,52 @@
+#
+# Copyright 2019-2020 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+Name:           grafana-ha-cluster-dashboards
+# Version will be processed via set_version source service
+Version:        0
+Release:        0
+License:        Apache-2.0
+Summary:        Grafana Dashboards displaying metrics about a Pacemaker/Corosync High Availability Cluster.
+Group:          System/Monitoring
+Url:            https://github.com/ClusterLabs/ha_cluster_exporter
+Source:         %{name}-%{version}.tar.gz
+BuildArch:      noarch
+Requires:       grafana
+BuildRequires:  grafana
```

Comment:
Is this required? I mean, to put some files in some folders during the build maybe grafana is not needed. If we can avoid it better hehe You know how the dependencies during the build time can be quite troubling

Reply:
I initially tried to do without it, but we require the whole directory structure put in place by the Grafana package.
I guess I could duplicate some of the spec statements, but then what defines what package owns what path? Wouldn't we have conflicts?

Reply:
I think is ok to require grafana, `BuildRequires` maybe we can spare it

Reply:
the build will fail without that `BuildRequire` with the following error:
```
[   28s] grafana-ha-cluster-dashboards-1.0.0+git.1591970022.e33fbe9-1.1.noarch.rpm: directories not owned by a package:
[   28s]  - /etc/grafana
[   28s]  - /etc/grafana/provisioning
[   28s]  - /etc/grafana/provisioning/dashboards
[   28s]  - /var/lib/grafana
[   28s]  - /var/lib/grafana/dashboards
```